lib/locales/en.ts

```diff
@@ -211,6 +214,7 @@ export const locale: Locale = {
     anthony_of_saint_anne_galvao_priest: 'Saint Anthony of Saint Anne Galvão, Priest',
     anthony_of_the_caves_monk: 'Saint Anthony of the Caves, Monk',
     anthony_zaccaria_priest: 'Saint Anthony Zaccaria, Priest',
+    antoine_chevrier_priest: 'Blessed Antoine Chevrier, Priest',
```

Comment:
Anglicise his first name ([src](https://leprado.org/wp-content/uploads/2018/12/Mass-of-F.Chevrier.pdf)).

```suggestion
    anthony_chevrier_priest: 'Blessed Anthony Chevrier, Priest',
```